platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/editor/StateEditor.java

```diff
@@ -62,7 +63,8 @@ public class StateEditor {
     @SuppressWarnings("java:S106")
     public StateEditor(final Path statePath) throws IOException {
 
-        final Configuration configuration = DefaultConfiguration.buildBasicConfiguration();
+        final Configuration configuration =
+                DefaultConfiguration.buildBasicConfiguration(getAbsolutePath("settings.txt"));
```

Comment:
We use `DefaultConfiguration.buildBasicConfiguration(getAbsolutePath("settings.txt"))` all over the place. That should be a direct method in `DefaultConfiguration`.

Reply:
method with default location added